contracts/InitializerAuRa.sol

```diff
@@ -16,6 +16,7 @@ contract InitializerAuRa {
         uint256 _stakingEpochDuration,
         uint256 _collectRoundLength
     ) public {
+        require(block.number == 0);

```

Comment:
This is excess because already done here: https://github.com/poanetwork/pos-contracts/blob/cfa8c7c65f4529725640c4734fb79bae7c230f94/contracts/abstracts/ValidatorSetBase.sol#L499